ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,243 @@
+| blockNumber       | QUANTITY        | Block number where this transaction was in.                                                |
+| cumulativeGasUsed | QUANTITY        | The total amount of gas used when this transaction was executed in the block.              |
+| effectiveGasPrice | QUANTITY        | The sum of the base fee and tip paid per unit of gas.                                      |
+| gasUsed           | QUANTITY        | The amount of gas used by this specific transaction alone.                                 |
+| contractAddress   | DATA, 20 Bytes  | The contract address created, if the transaction was a contract creation, otherwise `null` |
+| logs              | ARRAY           | Array of log objects, which this transaction'S EXECUTION FRAME generated.                  |
+| logsBloom         | DATA, 256 Bytes | Bloom filter for light clients to quickly retrieve related logs.                           |
+| type              | QUANTITY        | Integer of the transaction type                                                            |
+| status            | QUANTITY        | Either 1 (success) or 0 (failure) status of the execution frame                            |
+
+### Create a new JSON-RPC API method `eth_executeRip7560Transaction`
+
+Executes the entire RIP-7560 transaction in memory without broadcasting it or including it in a block.
+Does not require the transaction to be properly signed, meaning it continues execution after either an account
+or a paymaster contract make a `sigFailAccount` or `sigFailPaymaster` call.
+If all frames execute successfully, simply returns the data returned by the top level frame of the execution phase.
+If any of the validation or execution frames revers, returns an error object containing the revert message.
+Note that unlike `eth_call`, RIP-7560 transaction depends on its deployment and validation,
+so it can't be called in isolation from its deployment and validation
+If the transaction validation fails for any reason other than the failed signature check,
+returns an error object containing the details of the validation failure.
+
+Parameters:
+
+1. OBJECT - The RIP-7560 transaction object.
+   The `signature` field is optional.
+2. QUANTITY | TAG - integer block number, or the string "latest", "earliest", "pending", "safe" or "finalized"
+
+Returns:
+
+DATA - The return value of the `sender` execution frame.
+
+Error:
+
+DATA - The revert data of the first reverted frame.
+CODE - The error code indicating the type of error, which may include the entity that caused the revert on-chain.
+MESSAGE - The human-readable error that may include a decoding of the `DATA` field if possible.
+
+### Add RIP-7560 support for all remaining transaction-level RPC APIs
+
+This includes the following APIs: `eth_sendTransaction`, `eth_sendRawTransaction`,  `eth_getTransactionByHash`,
```

Comment:
How would `eth_sendTransaction` work?  The node is unaware of the account implementation and cannot generate a valid transaction for it.  It exists in clients because historically people ran nodes and kept keys in their keystores, unlocking the account and transacting with it.  Nowadays I think it's mostly used between the wallet and the dapp.  Will the node implement it in some way?

Reply:
In this case I thought it would basically mirror the `eth_sendRawTransaction` but without having to make an RLP encoding first. Meaning, all fields must be filled in, including `signature` a.k.a. `authorizationData`.
I would love us to define a dapp-wallet API as well but we would need to dedicate a lot of time to do it. As far as I understand, there isn't a final design for ERC-4337 as well, is that correct?

Reply:
I don't know if the ability to send an AA transaction as a JSON instead of an RLP-encoded byte array is worth adding a new API.
What do you think? Maybe, I should better remove the changes to `eth_sendTransaction` completely?